Dockerfile

```diff
@@ -117,6 +117,9 @@ RUN if [ "$TARGETPLATFORM" = "linux/amd64" ] && [ "$ONNX_AMD64_WHEEL" != "unknow
         pip install --no-cache-dir onnxruntime-gpu --extra-index-url https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-cuda-12/pypi/simple/ ; \
     fi
 
+# Install earth2grid
+RUN pip install --no-build-isolation https://github.com/NVlabs/earth2grid/archive/f1d534bf16870bf12ef1a2d9b13ee5de65d869b7.tar.gz
+
```

Comment:
I have an earth2grid build (from github main) in the [Dockerfile for NV-dlesm](https://github.com/AtmosSci-DLESM/NV-dlesm/blob/d56f6a59fd8458185a83d3dadf9a9d89b4cf353a/docker/Dockerfile#L23) FYI. This is fine to have here but when we upstream this I'm not sure if earth2grid will make it into the official physicsnemo image. I would be in favor but there are sometimes unexpected issues (e.g. low security ratings for external deps)

Reply:
I would be in favor of keeping this here and removing it when we eventually do an upstream PR, but this is a minor detail so removed e2grid build from this Dockerfile